packages/patterns/src/components/partial-syncing-controls.js

```diff
@@ -18,51 +18,45 @@ import { PARTIAL_SYNCING_SUPPORTED_BLOCKS } from '../constants';
 function PartialSyncingControls( { name, attributes, setAttributes } ) {
 	const syncedAttributes = PARTIAL_SYNCING_SUPPORTED_BLOCKS[ name ];
```

Comment:
Not sure if this PR would be the best place to do this, but I think we should rename this to `editableAttributes` or something similar for clarity, as these are the attributes that users will be able to override.